sormas-api/src/main/java/de/symeda/sormas/api/event/EventSelectionDto.java

```diff
@@ -0,0 +1,36 @@
+package de.symeda.sormas.api.event;
+
+import java.io.Serializable;
+
+import de.symeda.sormas.api.utils.pseudonymization.PseudonymizableIndexDto;
+
+public class EventSelectionDto extends PseudonymizableIndexDto implements Serializable, Cloneable {
```

Comment:
you can use EventReferenceDTO instead of creating this similar class

Reply:
indeed can be used, but if new requirements on what the Event labels should contain will increase then the EventSelectionDto looks more appropriate.

Reply:
We can create the more appropriate data structure when that requirement comes in.

BTW also the naming is misleading because it is not used for selection of events right now